packager/media/formats/mp4/box_definitions_unittest.cc

```diff
@@ -761,16 +761,6 @@ class BoxDefinitionsTestGeneral : public testing::Test {
 
   void Modify(Movie* moov) { moov->tracks.resize(1); }
 
-  void Fill(TrackFragmentDecodeTime* tfdt) {
-    tfdt->decode_time = 234029673820ULL;
-    tfdt->version = 1;
-  }
-
-  void Modify(TrackFragmentDecodeTime* tfdt) {
-    tfdt->decode_time = 4567;
-    tfdt->version = 0;
-  }
-
```

Comment:
This is not a dead code. It is used by the TYPED_TEST_P below to make sure tfdt can be written and read back correctly.